lib/utils/index.ts

```diff
@@ -36,9 +36,7 @@ export const getCleanType = (abiType: string, toLower = true) => {
   const isOptional = abiType.toLowerCase().startsWith("option");
   let cleanType = isOptional ? (abiType.match(/<(.*)>/) || [])[1] : abiType;
 
-  cleanType = cleanType;
-
-  cleanType = cleanType.split("<")[0];
+  if (isOptional) cleanType = cleanType.split("<")[0];
```

Comment:
if isOptional check is duplicated. Also, if you are cleaning the Type, i would re-use the same variable because Javascript is pass by value on strings
Suggestion:
```JavaScript
if (isOptional) {
    let matches = abiType.match(/<(.*)>/)
    if (matches && matches.length > 1) abiType = matches[1]  // prevents previous error of getting 1 of an empty array
    abiType = abiType .split("<")[0]; // index 0 is always returned, even with no match
}
```